library/core/src/marker.rs

```diff
@@ -540,7 +552,7 @@ macro_rules! impls {
 /// For a more in-depth explanation of how to use `PhantomData<T>`, please see
 /// [the Nomicon](../../nomicon/phantom-data.html).
 ///
-/// # A ghastly note 👻👻👻
+/// # A ghastly note 
```

Comment:
Any reason this was removed?
```suggestion
/// # A ghastly note 👻👻👻
```

Reply:
Oh no, why has it removed the cute emojis? Thanks for catching that. I'm really sorry. :cry: 
I can only imagine that it has something to do with my EmojiOne extension for Firefox.